FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Ke Ma on 11/09/2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct FeedImageMapper {
```

Comment:
`FeedImageMapper` is defined as a struct, so we can instantiate it as FeedImageMapper(). However, all its methods are static - it has no instance methods or properties.

It may be confusing for other developers to be able to instantiate an instance but then not be able to do anything with it.

As you'll see in the following lectures, you can make the initializer private in such cases since the `FeedImageMapper` is just a namespace for static methods, with no instance-specific behavior.

Or you can make it an enum with no cases, which can never be instantiated!

Both solutions will disallow instantiation at compile time so that `FeedImageMapper()` won't compile

```suggestion
enum FeedImageMapper {
```